third_party/crypto/acme/acme.go

```diff
@@ -73,6 +77,11 @@ type Client struct {
 	// will have no effect.
 	DirectoryURL string
 
+	// UserAgent is an optional string that identifies this client and
+	// version to the ACME server. It should be set to something like
+	// "myclient/1.2.3".
+	UserAgent string
```

Comment:
With the addition of this field, we should be able to remove the custom user agent code with currently have in the form of a HTTP RoundTripper.

For now, I think the round tripper should overwrite any value set by this package, meaning merging this PR will cause no change. We should add a unit test to this PR to verify that however.

Switching to use this field will cause a change to our UA by the looks of the implementation of `newRequest` (it wil be prepended with `go-acme/2`).

cc @jsha @cpu (as this may effect any metrics you already have)

Reply:
/hold

Reply:
Thanks for the tag @munnerz. I'll make sure we adjust any queries on our side :+1: